third-party/qthread/qthread-1.10/config/config.sub

```diff
@@ -238,6 +238,7 @@ case $basic_machine in
 	# Some are omitted here because they have special meanings below.
 	1750a | 580 \
 	| a29k \
+	| aarch64 | aarch64_be \
```

Comment:
Were the changes in this file hand edits or done by a script as part of a build?  Usually we don't want to change anything beyond the top-level files in third-party packages.  I'm not enough of a configure person to know if this is a good idea, but if it is we should try to get the qthreads team to do this in their next release so we don't have to do it ourselves.